framework/src/base/MooseObject.C

```diff
@@ -44,7 +44,6 @@ MooseObject::validParams()
   params.addPrivateParam<SubProblem *>("_subproblem", nullptr);
   params.addPrivateParam<SystemBase *>("_sys", nullptr);
   params.addPrivateParam<SystemBase *>("_nl_sys", nullptr);
-  params.addPrivateParam<AuxiliarySystem *>("_aux_sys", nullptr);
```

Comment:
In the past `FEProblemBase` was assigning the undisplaced auxiliary system as `_aux_sys` even when an auxiliary kernel was acting on the displaced mesh. This was wrong and lead to problems earlier in the PR. So I've removed this parameter to avoid the possibility of people doing this stupid thing again. The assignment of `_sys` has always been correct

Reply:
Great. I'd have definitely fallen for that.